src/index.tsx

```diff
@@ -24,6 +24,8 @@ import { readHistoryData } from './app/views/sidebar/history/history-utils';
 import messages from './messages';
 import { store } from './store';
 import './styles/index.scss';
+import { telemetry } from './telemetry';
```

Comment:
And also, this telemetry class is meant to be used in GE v3 as well.

Reply:
For Angular in GEv3, the logging pattern is to use a service. We should check with observability about the use of https://www.npmjs.com/package/@dagonmetric/ng-log-applicationinsights.

Reply:
Nevermind, there is this: https://github.com/microsoft/ApplicationInsights-JS/tree/master/extensions/applicationinsights-angular-js. I don't think we use much routing in GEv3 so it may make more sense to just use the base AppInsights library.